src/pages/Popup/components/Settings.jsx

```diff
@@ -123,6 +131,9 @@ class Settings extends Component {
       chrome.storage.sync.get(resolve)
     );
     await this.setState({ buttonDisabled: true, alarmData: Newresponse });
+    await this.setState({
```

Comment:
Add the background state in the above `this.setState` function itself. No need to write it again

Reply:
Updated.